recipe/meta.yaml

```diff
@@ -144,13 +146,18 @@ outputs:
         - iminuit >=2
         - lmfit
         - lsstdesc.coord
+        - lsst-ts-xml
         - matplotlib-base !=3.6.0
+        - mkl  # [linux64]
```

Comment:
Are we fixing to mkl on linux?  (May be fine, but not necessary I don't think).

Reply:
We have it in 7.0.1, so was just carrying it over.  Remove it?

Reply:
This was required because of the psfex problem. It should be optional and conda forge controllable I think